ios/RNCallKeep/RNCallKeep.m

```diff
@@ -188,11 +196,15 @@ + (void)initCallKitProvider {
     }
 }
 
-+ (NSString *) getAudioOutput {
++ (NSMutableDictionary *) getAudioOutput {
```

Comment:
I see the contents here differ slightly from our patch -- should I assume you created the fork against the latest main branch on the upstream repo, and there must be some new changes there that we didn't pick up yet?  
The differences look like they should be fine, but let's make sure we re-test all this stuff...

Reply:
Correct, the main thing they added was a try/catch here:
![image](https://github.com/streem/react-native-callkeep/assets/425965/3ec7aad2-eb02-4209-a307-3b1b7256f41c)